pullrequest.rst

```diff
@@ -119,10 +119,30 @@ homepage.
 Submitting
 ----------
 
-Once you are satisfied with your work you will want to commit your
-changes to your branch. In general you can run ``git commit -a`` and
-that will commit everything. You can always run ``git status`` to see
-what changes are outstanding.
+Once you are satisfied with your work, you will need to commit your
+changes to your branch. Assume you want to add a file which you have changed,
+you will need to enter this command::
```

Comment:
Apparently I assumed this worked like Mercurial, but it doesn't...

So `git add` doesn't mean "add this file to the tracked files starting from the next commit", but something like "add this file to the list of files that will get committed when I do `git commit`), right?
Also will `git commit -am` include all the modified files, including new files that were previously untracked?